docs/src/main/sphinx/release/release-411.md

```diff
@@ -0,0 +1,59 @@
+# Release 411 (15 Mar 2023)
+
+## General
+
+* Add spilled data size to query statistics. ({issue}`16442`)
+* Add {func}`sinh` function. ({issue}`16494`)
+* Add support for a `GRACE PERIOD` clause in the `CREATE MATERIALIZED VIEW`
+  task. This is a backwards incompatible change, and the previous behavior can
+  temporarily be restored with the `legacy.materialized-view-grace-period`
+  configuration property or the `legacy_materialized_view_grace_period` session
+  property. ({issue}`15842`)
+* Add `CANCELING`, `ABORTING`, and `FAILING` task statuses for tasks which are
+  waiting on running drivers to fully terminate. ({issue}`15478`)
+* Improve error messages for exceptions thrown by a connector while querying
+  metadata. ({issue}`16497`)
+* Fix generation of IDs for `keyvalue()` methods in the
+  [JSON path](json-path-language). Previously, separate IDs were generated for
+  each member. ({issue}`16482`)
+
+## Delta Lake connector
+
+* Add support for inserting into tables that have
+  [column invariants](https://github.com/delta-io/delta/blob/master/PROTOCOL.md#column-invariants). ({issue}`16136`)
+* Fix query failure when evaluating a `WHERE` clause on a partition column. ({issue}`16388`)
+
+## Elasticsearch connector
+
+* Remove the deprecated pass-through query, which has been replaced with the
+  `raw_query` table function. ({issue}`13050`)
+
+## Hive connector
+
+* Improve performance of queries with joins where both sides of a join have join
+  keys with the same table bucketing definition. ({issue}`16381`)
+* Improve query planning performance for queries scanning tables with a large
+  number of columns. ({issue}`16203`)
+* Fix query failure when a file is split when using a text file format with a
```

Comment:
can we reword .. double "when" sounds weird

Reply:
maybe 

when a file, that is using a text format with a single header row, is split.